src/libraries/Microsoft.Extensions.Configuration/src/ConfigurationSection.cs

```diff
@@ -69,11 +69,11 @@ public ConfigurationSection(IConfigurationRoot root, string path)
         {
             get
             {
-                return _root[ConfigurationPath.Combine(Path, key)];
+                return _root[Path + ConfigurationPath.KeyDelimiter + key];
```

Comment:
Here are three places where the lookup-key is created. Although it's trivial code, should this be moved to a helper method in order to have only one place where that lookup-key is built?

Reply:
Ideally, should add an overload to `ConfigurationPath.Combine` that takes two path strings (the most common case), but adding it as an internal API wouldn't work because it's in a different assembly and adding a public API needs the full API review process. I'm willing to create an API review request for this, if it sounds useful enough?

Reply:
I don't think a public API is needed here. I thought just a helper in `ConfigurationSection` (as the PR's diff shows three places where that could be used).